spark/src/main/scala/org/apache/spark/sql/delta/DeltaOperations.scala

```diff
@@ -471,6 +471,8 @@ object DeltaOperations {
 
   sealed abstract class OptimizeOrReorg(override val name: String, predicates: Seq[Expression])
     extends OperationWithPredicates(name, predicates)
+  /** parameter key to indicate whether it's an Auto Compaction */
```

Comment:
to indicate Auto Compaction

Reply:
re https://github.com/delta-io/delta/pull/2414#issuecomment-1883922342, perhaps we can address post merge